src/vivarium_gates_iv_iron/data/utilities.py

```diff
@@ -237,3 +234,13 @@ def get_gbd_age_bins(age_group_ids: List[int] = None) -> pd.DataFrame:
     # set age start for birth prevalence age bin to -1 to avoid validation issues
     age_bins.loc[age_bins['age_end'] == 0.0, 'age_start'] = -1.0
     return age_bins
+
+
+def get_truncnorm_from_quantiles(mean: float, lower: float, upper: float,
+                                 quantiles: Tuple[float, float] = (0.025, 0.975),
+                                 lower_clip: float = 0.0, upper_clip: float = 1.0) -> stats.truncnorm:
+    stdnorm_quantiles = stats.norm.ppf(quantiles)
+    sd = (upper - lower) / (stdnorm_quantiles[1] - stdnorm_quantiles[0])
+    a = (lower_clip - mean) / sd if sd else 0.0
+    b = (upper_clip - mean) / sd if sd else 0.0
```

Comment:
should this read "else 1"?

Reply:
I may not be following the arguments fully, but just wanted to check! Otherwise looks good to me

Reply:
This should actually be like this:

    a = (lower_clip - mean) / sd if sd else mean - 1e-03
    b = (upper_clip - mean) / sd if sd else mean + 1e03

Reply:
Awesome, thanks @alibow and @rmudambi. I'll make this change.

Reply:
Updated this per discussion at checkin -- if we have sd == 0, then that implies upper == lower, so we just use a scalar (a normal distributed centered at the mean and with sd=0).